src/tests/suite/daos_dedup.c

```diff
@@ -231,6 +245,13 @@ with_identical_updates(void *const *state, uint32_t iod_type, int csum_type,
 	daos_size_t		delta;
 	int			rc;
 
+	if (dedup_type == DAOS_PROP_CO_DEDUP_MEMCMP &&
+	    dedup_is_nvme_enabled(*state)) {
+		print_message("DAOS_PROP_CO_DEDUP_MEMCMP "
+			      "doesn't support NVMe.\n");
+		skip();
```

Comment:
If you skip only the MEMCMP cases, the get_size() needs be improved to check NVMe free size when NVMe is enabled. (Seems the dedup threshold is larger than 4k in all dedup tests, so update will be landed to NVMe when NVMe is enabled, and we need to check NVMe free space respectively).

Reply:
okay, I'll just skip for all dedup for now. :-)